src/generators/go/GoConstrainer.ts

```diff
@@ -48,6 +69,31 @@ export const GoDefaultTypeMapping: GoTypeMapping = {
   }
 };
 
+function getType({
+  constrainedModel,
+  partOfProperty,
+  typeWhenNullableOrOptional,
+  type
+}: {
+  constrainedModel: ConstrainedMetaModel;
+  partOfProperty: ConstrainedObjectPropertyModel | undefined;
+  typeWhenNullableOrOptional: string;
+  type: string;
+}) {
+  let requried = false;
+  if (partOfProperty) {
+    if (partOfProperty.required) {
+      requried = partOfProperty.required;
+    } else {
+      requried = false;
+    }
+  }
```

Comment:
Do you mind simplifying this if statement?  - and its called `required` not `requried` 🙂